test/test_integration_credentials.py

```diff
@@ -211,30 +217,32 @@ def test_https_netrc_is_created_with_host(self):
         output = self.invoke_cli(self.cli_auth_params,
                                  ['login', '-i', 'host/somehost', '-p', f'{extract_api_key_from_message}'])
 
-        self.validate_netrc(f"{self.client_params.hostname}", "host/somehost", extract_api_key_from_message)
+        self.validate_credentials(f"{self.client_params.hostname}", "host/somehost", extract_api_key_from_message)
         self.assertIn("Successfully logged in to Conjur", output.strip())
 
+    @integration_test()
+    def test_working_with_keystore(self):
```

Comment:
I think the following name is more descriptive of what you are testing here
```suggestion
    def test_provider_can_return_keystore_provider(self):
```

Reply:
same for file netrc tests